content/docs/5.1/develop/clients/java.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Java Client
+description: Interact with TiKV using Java.
+menu:
+    "5.1":
+        parent: TiKV Clients
+        weight: 1
+---
+
+This guide introduces how to interact with TiKV using [Java Client](https://github.com/tikv/client-java).
+
+{{< warning >}}
+TiKV Java Client is developed and released using Java8. The minimum supported version of TiKV is 2.0.0.
+{{< /warning >}}
+
+## Add the dependency
+
+To start, open the `pom.xml` of your project, and add the `tikv-client-java` as dependencies if you are using Maven.
+
+```xml
+<dependency>
+  <groupId>org.tikv</groupId>
+  <artifactId>tikv-client-java</artifactId>
+  <version>${latest_version}</version>
+</dependency>
+```
+
+## Raw key-value API
```

Comment:
it's better to illustrate the client-java in these parts:

- a quick introduction and demo show
- the data insertion/deletion operations
- the data retrieving operations
- atomic operations

in this document, we illustrate the functionalities that the java client can provide, helps users to understand what they can do with this client, leave the concert API description to the API Doc.

Reply:
I will put the examples in `Develop -> RawKV` and `Develop -> TxnKV` chapter.